odoo/tools/test_reports.py

```diff
@@ -41,21 +41,20 @@ def try_report(cr, uid, rname, ids, data=None, context=None, our_module=None, re
     if res_format == 'pdf':
         if res_data[:5] != b'%PDF-':
             raise ValueError("Report %s produced a non-pdf header, %r" % (rname, res_data[:10]))
-        res_text = False
+        res_text = None
         try:
             fd, rfname = tempfile.mkstemp(suffix=res_format)
             os.write(fd, res_data)
             os.close(fd)
 
-            proc = Popen(['pdftotext', '-enc', 'UTF-8', '-nopgbrk', rfname, '-'], shell=False, stdout=PIPE)
-            stdout, stderr = proc.communicate()
-            res_text = ustr(stdout)
+            proc = Popen(['pdftotext', '-enc', 'UTF-8', '-nopgbrk', rfname, '-'], shell=False, stdout=PIPE, encoding="utf-8")
```

Comment:
shell=False is the default value
dito `text=True` instead of `encoding='utf-8'`, for windaube